src/common/SourceFilter/SourceFilter.js

```diff
@@ -1,16 +1,14 @@
 import React from 'react';
 
-import OrdersSelectionFilter from '../OrdersSelectionFilter';
+import { sourceOptions } from '@folio/stripes-acq-components/util/constants';
 
-const SourceFilter = (props) => {
-  const options = [{ value: 'FOLIO', label: 'FOLIO' }];
+import OrdersSelectionFilter from '../OrdersSelectionFilter';
```

Comment:
Since it's enum, I don't think it should be a Select. Maybe it's better to make this as Checkbox filter

Reply:
Fixed